src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+    <div class="gallery-text">
+        <!-- Title -->
+        <p class="title">Our Ecosystem</p>
+
+        <!-- Subtitle w/ Arrow -->
+        <p class="subtitle">
+            <span class="subtitle-text">We develop and produce original content for all genres and formats.</span>
+            <span class="prompt">
+                <p class="prompt-text">EXPLORE</p>
+                <span class="arrow">Arrow</span>
```

Comment:
When this text wraps, the arrow should be at the end of the sentence, not over to the right as it is now. See screenshot.

![Screen Shot 2020-07-02 at 9 06 55 PM](https://user-images.githubusercontent.com/475015/86430798-fdb0bc80-bca7-11ea-9226-b05295a72946.png)